config/Config-build.in

```diff
@@ -386,4 +386,15 @@ menu "Global build settings"
 
 	endchoice
 
+	config APPARMOR
+		bool "Enable AppArmor"
+		select KERNEL_SECURITY_APPARMOR
+		select KERNEL_AUDIT
```

Comment:
Is audit always required or is it just a nice to have feature?

Reply:
audit is requirement.

Reply:
Please, remove this change request.